src/routes/Settings/utils/formatters.js

```diff
@@ -0,0 +1,23 @@
+// @flow
+
+export function formatBoolean(value: boolean): string {
+  return value ? 'Enabled' : 'Disabled'
+}
+
+export function formatLanguage(langCode: string): string {
+  const languageMap = {
+    en: 'English',
+    kr: 'Korean',
+  }
+
+  return languageMap[langCode] || 'Language not found'
+}
+
+export function formatCurrency(curCode: string): string {
+  const currencyCode = {
+    usd: 'US Dollar',
+    rub: 'Russian Ruble',
+  }
```

Comment:
we should use such languageMap/currencyCode maps in base config.js or in `src/data` dir